burrow/templates/deployment.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "burrow.fullname" . }}
+  labels:
+    {{- include "burrow.labels" . | nindent 4 }}
+  annotations:
+    {{- toYaml .Values.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "burrow.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ tpl (toYaml .Values.config) . | sha256sum }}
```

Comment:
@cw-sakamoto 
This is a suggestion.
Wouldn't it be better to use `sha256` with include?

`toYaml` is not recommended due to long lines that cause incorrect wrapping behavior.
https://v2.helm.sh/docs/developing_charts/